models/chain_swap/arbitrum/chain_swap_arbitrum_trades.sql

```diff
@@ -0,0 +1,143 @@
+{{
+    config(
+        alias='trades',
+        schema="chain_swap_arbitrum",
+        partition_by=["block_month"],
+        materialized="incremental",
+        file_format="delta",
+        incremental_strategy="merge",
+        incremental_predicates=[
+            incremental_predicate("DBT_INTERNAL_DEST.block_time")
+        ],
+        unique_key=["blockchain", "tx_hash", "evt_index"],
+    )
+}}
+
+{% set project_start_date = "2024-03-19" %}
+{% set blockchain = "arbitrum" %}
+{% set deployer_1 = "0x9eC1ACAe39d07E1e8D8B3cEbe7022790D87D744A" %}
+{% set deployer_2 = "0x415EEc63c95e944D544b3088bc682B759edB8548" %}
+{% set deployer_3 = "0xc1cc1a300Dcfe5359eBe37f2007A77d1F91533ba" %}
+{% set deployer_4 = "0x3A510C5a32bCb381c53704AED9c02b0c70041F7A" %}
+{% set deployer_5 = "0xb7b953e81612c57256ff0aebd62b6a2f0546f7da" %}
+{% set weth_contract_address = "0x82aF49447D8a07e3bd95BD0d56f35241523fBab1" %}
+{% set usdc_contract_address = "0xaf88d065e77c8cC2239327C5EDb3A432268e5831" %}
+{% set fee_recipient_1 = "0x415EEc63c95e944D544b3088bc682B759edB8548" %}
+{% set fee_recipient_2 = "0xe1ff5a4c489b11e094bfbb5d23c6d4597a3a79ad" %}
+
+with
+    bot_contracts as (
+        select address
+        from {{ source("arbitrum", "creation_traces") }}
+        where
```

Comment:
Please use single quotes (') in the code as much as possible. Double quotes (") can sometimes break the code.